soc/arm/nuvoton_npcx/Kconfig

```diff
@@ -20,6 +20,27 @@ menuconfig NPCX_HEADER
 
 if NPCX_HEADER
 
+config NPCX_IMAGE_NAME
+	string "The npcx image binary name"
+	default "npcx_image"
+	help
+	  This option sets the name of the generated npcx binary.
+
+config NPCX_IMAGE_OUTPUT_BIN
+	bool "Build npcx binary in BIN format"
+	default y
+	help
+	  Build a "raw" binary zephyr/npcx_image.bin in the build directory.
+	  The name of this file can be customized with CONFIG_NPCX_IMAGE_NAME.
+
+config NPCX_IMAGE_OUTPUT_HEX
+	bool "Build npcx binary in HEX format"
+	depends on NPCX_IMAGE_OUTPUT_BIN
+	help
+	  Build an HEX binary zephyr/npcx_image.hex in the build directory.
+	  This generates from npcx BIN image.
```

Comment:
This is generated from the npcx BIN image

Reply:
Done.